src/test_l1_tvSettings.c

```diff
@@ -1223,20 +1215,13 @@ void test_l1_tvSettings_positive_GetCurrentVideoResolution (void)
 	result = GetCurrentVideoResolution(&tvVideoResolution);
 	UT_ASSERT_AUTO_TERM_NUMERICAL(result, tvERROR_NONE);
 
-	if(  (tvVideoResolution.resolutionValue > tvVideoResolution_NONE)  && (tvVideoResolution.resolutionValue < tvVideoResolution_MAX) && \
-			(tvVideoResolution.frameHeight > 0) && (tvVideoResolution.frameWidth > 0) 	) {
-		IsVideoResolutionValid = true;
-	} 
-	UT_ASSERT_AUTO_TERM_TRUE((IsVideoResolutionValid));
+	UT_ASSERT_EQUAL(tvVideoResolution.resolutionValue,tvVideoResolution_NONE);
 
 	/* Step 03: Calling tvsettings GetCurrentVideoFormat and expectinging the API to return success */
 	result = GetCurrentVideoResolution(&tvVideoResolutionRetry);
 	UT_ASSERT_AUTO_TERM_NUMERICAL(result, tvERROR_NONE);
 
-	UT_ASSERT_AUTO_TERM_NUMERICAL(tvVideoResolutionRetry.resolutionValue, tvVideoResolution.resolutionValue);
-	UT_ASSERT_AUTO_TERM_NUMERICAL(tvVideoResolutionRetry.frameHeight, tvVideoResolution.frameHeight);
-	UT_ASSERT_AUTO_TERM_NUMERICAL(tvVideoResolutionRetry.frameWidth, tvVideoResolution.frameWidth);
-	UT_ASSERT_AUTO_TERM_NUMERICAL(tvVideoResolutionRetry.isInterlaced, tvVideoResolution.isInterlaced);
+	UT_ASSERT_EQUAL(tvVideoResolutionRetry.resolutionValue, tvVideoResolution.resolutionValue);
```

Comment:
test_l1_tvSettings_positive_GetCurrentVideoFrameRate  - Please modify the step 2 in the description as we are now reading the default out parameter. 

| 02 | call GetCurrentVideoFrameRate() -  Retrieve the current VideoFrameRate and validate VideoFrameRate by looping through the VideoFramerate section of test specific config file values

Similar way, please check for the other test functions description.

Reply:
updated.